pkg/setting/setting.go

```diff
@@ -1225,3 +1232,9 @@ func readServerSettings(iniFile *ini.File, cfg *Cfg) error {
 
 	return nil
 }
+
+func readDataSourcesSettings(iniFile *ini.File, cfg *Cfg) error {
```

Comment:
See previous comment, also there's no error to return:
```suggestion
func (cfg *Cfg) readDataSourceSettings() {
```